openfisca_core/tools/test_runner.py

```diff
@@ -110,12 +110,13 @@ def run_tests(tax_benefit_system, paths, options = {}):
     +-------------------------------+-----------+-------------------------------------------+
 
     """
+    argv = sys.argv[:1]  # Nose crashes if it gets any unexpected argument.
+    if options.get('verbose'):
```

Comment:
Is this documented somewhere ?

Reply:
Yep, but it was broken. Will be repaired by this PR.

http://openfisca.readthedocs.io/en/trace/test_runner.html
http://openfisca.readthedocs.io/en/trace/openfisca-run-test.html